rcl/include/rcl/timer.h

```diff
@@ -137,6 +137,7 @@ RCL_WARN_UNUSED
 rcl_ret_t
 rcl_timer_init(
   rcl_timer_t * timer,
+  rcl_clock_t * clock,
```

Comment:
docblock for this function needs to be updated

Reply:
Absolutely, packages on-top of `rclcpp` also haven't been updated yet. Will put it back in review when that is the case and the docblocks are updated too.

Reply:
Updated the docblock in e1a400e6724de65cab6969c90fd798675b144fbb.